### PR TITLE
update links

### PR DESCRIPTION
**Pull request by scottrepreneur**
- expand explorer links
- expand safe links

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ const Home: NextPage = () => {
 						<CardBody>
 							<VStack gap="25px" alignItems={"flex-start"}>
 								<Button
-									isDisabled={!isReadyToUse}
+									isDisabled // ={!isReadyToUse}
```
Comment (birimbau) [inline pages/index.tsx:109]: why this?

File: components/HeaderActions.tsx [hunk 2]
```diff
@@ -29,8 +29,8 @@ const HeaderActions: React.FC<P> = (p) => {
 				Deploy
 			</Button>
 
-			<Button
-				isDisabled={!isReadyToUse}
+			{/* <Button
```
Comment (birimbau) [inline components/HeaderActions.tsx:32]: Why this buttons are comented?

**Review by birimbau — CHANGES_REQUESTED** [review]
Check comments

File: pages/index.tsx (hunk 1 above)
Comment (scottrepreneur) [reply pages/index.tsx]: I reverted this extra commit. Please review with just the link updates. 🙏

**Review by birimbau — APPROVED** [review]

**Pull request merged**